packages/storage/src/service.ts

```diff
@@ -158,9 +159,9 @@ export class FirebaseStorageImpl implements FirebaseStorage {
    * This string can be in the formats:
    * - host
    * - host:port
-   * - protocol://host:port
    */
   private _host: string = DEFAULT_HOST;
+  protocol: string = 'https';
```

Comment:
Does this need to be exposed via the API?

Reply:
This whole class is internal so it shouldn't be exposed publicly. But it can't be private since `connectStorageEmulator` needs to be able to set it. I can add an underscore so it's clear that it's not meant to be public?

Reply:
The rest of the class does use underscores. I merely asked because of the change in the API report as it is not obvious that this is not exposed.

Reply:
Added an underscore.